Project.toml

```diff
@@ -1,17 +1,14 @@
 name = "CALCEPH"
 uuid = "1537fe66-4725-5aba-80f4-3a74792cecc1"
 authors = ["Bernard Godard"]
-version = "1.1.0"
+version = "1.2.0"
 
 [deps]
-BinDeps = "9e28174c-4ba2-5203-b857-d8d62c4213ee"
-BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
-Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
+CALCEPH_jll = "dd3044d3-6c85-52eb-b239-9fbeaa51d7ac"
 
 [compat]
-BinDeps = "^1"
-BinaryProvider = "^0"
-julia = "^1.0"
+julia = "1.3"
+CALCEPH_jll = "3.5"
```

Comment:
CALCEPH_jll v3.5 requires Julia v1.6, so you either use CALCEPH_jll v3.4 + julia v1.3 as [I had suggeste](https://github.com/JuliaAstro/CALCEPH.jl/pull/11#discussion_r1386977950) or you use CALCEPH_jll v3.5 + julia v1.6 :slightly_smiling_face:

Reply:
The older CALCEPH versions (< 3.5) have large errors in the computation of some SPK type 1/21 segments, that's why I'd move to the newest one. I've fixed the minimum julia version. 

I can make a separate pull request to update the CI aswell with the GitHub one!